src/components/stack/_index.scss

```diff
@@ -0,0 +1,27 @@
+@use '../../scss';
+
+.c-stack {
+	@include scss.component-properties('stack');
+	display: flex;
+}
+
+.c-stack[data-direction='vertical'] {
```

Comment:
The `data-*` attributes are a nice way to achieve what we are trying to do. As these attributes are for style. What if we set a precedence of using `data-style-*` to denote they are being used for styling purposes?

Reply:
Love it

Reply:
Done.